src/main/java/mate/controller/CreateDriverController.java

```diff
@@ -0,0 +1,38 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers/add")
+public class CreateDriverController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final DriverService driverService
+            = (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/drivers/createDriver.jsp").include(req, resp);
+        List<Driver> allDrivers = driverService.getAll();
+        req.setAttribute("drivers", allDrivers);
+        req.getRequestDispatcher("/WEB-INF/views/drivers/getAll.jsp").include(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String driversName = req.getParameter("Driver's name");
+        String driversLicenseNumber = req.getParameter("Driver's license number");
```

Comment:
```suggestion
        String driverName = req.getParameter("Driver's name");
        String driverLicenseNumber = req.getParameter("Driver's license number");
```
in other places you use singular and only here plural, you should keep your code consistent